cmake/BLTMacros.cmake

```diff
@@ -663,3 +663,56 @@ macro(blt_append_custom_compiler_flag)
 
 endmacro(blt_append_custom_compiler_flag)
 
+
+
+
+##------------------------------------------------------------------------------
+## blt_find_libraries( <VAR>
+##                     NAMES [libname1 [libname2 ...]]
+##                     PATHS [path1 [path2 ...]] )
+##
+## This command is used to find a list of libraries. A cache entry named by <VAR> 
+## is created to store the result of this command. If the libraries are found the
+## results are stored in VAR.
+##
+## NAMES lists the names of the libraries to search for.
+##
+## PATH lists the paths in which to search for NAMES.
+##
+##
+##------------------------------------------------------------------------------
+macro(blt_find_libraries VAR)
```

Comment:
While CMake does this on a regular basis, having unnamed vars causes problems with wrapping macros.  Please name the VAR variable, like OUTPUT or whatever you like.

Reply:
I renamed this to FOUND_LIBS.